.editorconfig

```diff
@@ -7,3 +7,10 @@ insert_final_newline = true
 
 [{*.json,*.yml,*.ts,*.tsx,*.md}]
 indent_style = space
+
+[*.md]
+indent_size = 2
+
+# Top-level .prettierrc.json uses tabs
+[/.prettierrc.json]
+indent_style = tab
```

Comment:
It's fine for specific libraries to use tabs if they feel like it, but let's stick with spaces for general DT files.

Reply:
But tabs are&nbsp;obviously the&nbsp;superior option, as&nbsp;you are looking at&nbsp;1&nbsp;byte per&nbsp;indentation&nbsp;level (tabs) vs&nbsp;several&nbsp;bytes per&nbsp;indentation&nbsp;level (spaces).